File: src/main/java/project/rpg/commands/FileSaveCommand.java
package project.rpg.commands;

import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.jetbrains.annotations.NotNull;
import project.rpg.manager.FileManager;

public class FileSaveCommand implements CommandExecutor {


    @Override
    public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, @NotNull String[] args) {
        //FileManager.makeFile();
        //FileManager.saveFile();
        FileManager.makeDir();
        FileManager.asyncSaveAll();
        //FileManager.getFile();

        return false;
    }
}

File: src/main/java/project/rpg/commands/FileSaveCommand.kt
package project.rpg.commands

import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender

class FileSaveCommand : CommandExecutor {
    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        TODO("JSON serialization not implemented yet")

        return false
    }
}